apps/lfx-one/src/app/shared/components/lens-switcher/lens-switcher.component.scss

```
@@ -72,3 +72,54 @@
   // stylelint-disable-next-line declaration-no-important
   margin-bottom: 0 !important;
 }
+
+// Create menu — same treatment as the LFX Apps menu (arrow hidden, no content
+// padding, pinned beside the rail). The Create button sits near the top of the
+// rail, so this panel is top-anchored and opens downward, mirroring how the
+// bottom-anchored apps menu opens upward from its button.
+::ng-deep .lfx-create-popover.p-popover {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)